### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by Havrysh**
[DEMO LINK](https://Havrysh.github.io/react_static-list-of-todos/)

### CODE REVIEW

File: src/components/TodoList/TodoList.scss [hunk 1]
```diff
@@ -0,0 +1,3 @@
+ul {
+  list-style-type: none;
+}
```
Comment (valerii-derkach) [inline src/components/TodoList/TodoList.scss:3]: check if you had new line in the end of file

**Review by valerii-derkach — CHANGES_REQUESTED** [review]
The structure is good!
Please provide a DEMO LINK

and add containers for your todo info, name the title, info, user, for better interface everything is rendering in a row, so it is unreadable

**Comment by Havrysh** [comment]
> 

Structure is your accomplishment)

**Review by valerii-derkach — APPROVED** [review]
https://www.loom.com/share/843e2c07431f48f3bc45c0cb86c775b6

**Pull request closed without merging**